### DISCUSSION TITLE
关于VirtualTileBox使用问题

### DISCUSSION THREAD

**Discussion by zzq158** [netease-im/NIM_Duilib_Framework#493] [Q&A]
在使用例子中virtualbox进行实现时，
 1、滚动鼠标滚轮不能丝滑的移动里面的item，会出现横跳的情况，是怎么回事呢？
 2、在显示的时候，最后会有多余的一行没有显示数据，这个该怎么解决呢？
![Snipaste_2024-01-15_11-18-18](https://github.com/netease-im/NIM_Duilib_Framework/assets/62682816/b04f0ecc-1d2c-431f-9c24-417a70bda78b)

**Comment by nmgwddj** [comment]
可以参考使用 duilib 开发的 IM Demo 中，有群组成员使用虚拟列表的能力实现可以作为参考：https://github.com/netease-kit/NIM_PC_Demo/tree/master/tool_kits/ui_component/ui_kit/gui/team_info
另外群成员列表也同样使用虚拟列表实现，可以作为参考.

**Reply by zzq158** [reply]
您好，Demo中依然没能够解决虚拟列表鼠标滚轮滚动时的item抖动问题，通过Debug源码，找到问题。LazyArrangeChild()方法中对每个item位置的设定应该减去滚轮所移动的位置，代码改动如图，此时可能会有自适应的item数目少一个的问题，可以将最大item数量增加一个。
![Uploading Snipaste_2024-01-18_12-03-54.png…]()
